doc/CodeDocumentation.conf.in

```diff
@@ -1125,7 +1125,13 @@ IMAGE_PATH             =
 # need to set EXTENSION_MAPPING for the extension otherwise the files are not
 # properly processed by doxygen.
 
-INPUT_FILTER           =
+# Allow the use of standard LaTeX formulas in MFEM comments by replacing
+# $$...$$ with \f[...\f] and $...$ with \f$...\f$ before running Doxygen.
+# The regular expression ((?:.|\n)+?) is a lazy match for one or more
+# characters, include newline. If present, Doxygen-style LaTeX commands
+# such as \f$, \f[, etc., are left unchanged.
+
+INPUT_FILTER           = perl -0777 -pe 's/\$\$((?:.|\n)+?)\$\$/\\f[\1\\f]/g; s/(?<!\\f)\$((?:.|\n)+?)\$/\\f\$\1\\f\$/g'
```

Comment:
I'm not sure all of the above (e.g. `(?<!\\f)`) is supported in `sed`. If it is, we can switch.

Reply:
Right, sed does not support many of those more advanced constructions. There are some workarounds for some (or maybe all?) of them. I played with that a while, but it would be too convoluted, so I am giving up 😏 Doxygen is not an essential part of the library, so who wants must pay the little toll of installing `perl`.

Reply:
At least on Linux and Mac perl is typically pre-installed.